pulp_rpm/app/modelresource.py

```diff
@@ -99,11 +107,18 @@ def set_up_queryset(self):
         """
         return PackageGroup.objects.filter(pk__in=self.repo_version.content)
 
+    related_packages = fields.Field(
+        column_name="related_packages",
+        attribute="related_packages",
+        widget=ManyToManyWidget(Package, ",", "pkgId"),
+    )
+
```

Comment:
All relations for comps types are removed, so I think this is not needed.

Reply:
correct - haven't revisited this since that change was merged. Updated and repushed.